modules/_labs/tabs/react/lib/TabPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {useTab} from './Tabs';
+
+export interface TabPanelProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The contents of the TabPanel.
+   */
+  children: React.ReactNode;
+  /**
+   * The name of the tab. This name will be used in change events and for `initialTab`. Must match
+   * the `name` of the associated tab item. If this property is not provided, it will default to a
+   * string representation of the the zero-based index of the Tab when it was initialized.
+   */
+  name?: string;
+}
+
+const TabPanel = ({children, name = '', ...elemProps}: TabPanelProps) => {
+  const {id, activeTab, registerPanel, unregisterPanel} = useTab();
+  const [tabName, setTabName] = React.useState(name);
+
+  // useLayoutEffect so we don't an incorrect frame if a name isn't provided
+  React.useLayoutEffect(() => {
+    const tabName = registerPanel(name);
+    setTabName(tabName);
+
+    return () => {
+      unregisterPanel(tabName);
+    };
+  }, [name, registerPanel, unregisterPanel]);
+
+  return (
+    <div
+      role="tabpanel"
+      aria-labelledby={`tab-${id}-${tabName}`}
+      hidden={!!tabName && tabName !== activeTab}
+      id={`tabpanel-${id}-${tabName}`}
+      tabIndex={0}
```

Comment:
@NicholasBoll what was the reason for making this focusable? If we intend to keep it this way, we need to add `InputProvider` handling

Reply:
👍

Reply:
Accessibility says it needs to be focusable. The alternative tab stop story shows how to override this functionality: https://github.com/Workday/canvas-kit/pull/569/files#diff-37fa226a19dc556892d7d02a377f11633ffba98f9052ca038665fd8a04720ca0R93-R111

I added `InputProvider` support to remove the focus outline when using the mouse

Reply:
Story in action: https://5d854c26ba934e0020f5e98a-utszliksxg.chromatic.com/?path=/story/labs-tabs-react--alternative-tab-stop